### PR TITLE
Selects

### PR DESCRIPTION
**Pull request by Kwolik**
Repair all selects and footer in adding work

### CODE REVIEW

File: web-client/src/components/PersonalInformation.jsx [hunk 1]
```diff
@@ -242,60 +245,48 @@ export default function PersonalInformation(props) {
...
+            <TextField
+              select
+              className={style.textField}
+              inputRef={register}
+              required
+              id="academic-title-signup"
               name="specialization"
-              className={style.formControl}
+              label="Specialization"
+              autoComplete="job-title"
```
Comment (upsu90) [inline web-client/src/components/PersonalInformation.jsx]: it isn't an academic title or job title so please change id and autoComplete

File: web-client/src/components/PersonalInformation.jsx [hunk 2]
```diff
@@ -242,60 +245,48 @@ export default function PersonalInformation(props) {
...
+                Mathematics               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Biology"}>
+                Biology               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Chemistry"}>
+                Chemistry               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Psychics"}>
+                Psychics
```
Comment (upsu90) [inline web-client/src/components/PersonalInformation.jsx]: please change to Physics

File: web-client/src/components/SignUpForm.jsx [hunk 3]
```diff
@@ -289,57 +292,46 @@ export default function SignUpForm(props) {
...
+                Mathematics               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Biology"}>
+                Biology               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Chemistry"}>
+                Chemistry               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Psychics"}>
+                Psychics
```
Comment (upsu90) [inline web-client/src/components/SignUpForm.jsx]: Physics :p

File: web-client/src/components/SignUpForm.jsx [hunk 4]
```diff
@@ -289,57 +292,46 @@ export default function SignUpForm(props) {
...
+            <TextField
+              select
+              className={style.textField}
+              inputRef={register}
+              required
+              id="academic-title-signup"
               name="specialization"
-              className={style.formControl}
+              label="Specialization"
+              autoComplete="job-title"
```
Comment (upsu90) [inline web-client/src/components/SignUpForm.jsx]: id and autoComplete again

File: web-client/src/pages/adding-work.js [hunk 5]
```diff
@@ -141,6 +142,10 @@ function AddingWork() {
         })
       )
       .max(maxAuthors, "You cannot add more authors"),
+    specialization: yup.string().when("specializations", {
+      is: (specializations) => specialization === "Select",
+      then: yup.string().required("Time field is required"),
```
Comment (upsu90) [inline web-client/src/pages/adding-work.js]: why "Time field"?

File: web-client/src/pages/adding-work.js [hunk 6]
```diff
@@ -357,58 +357,46 @@ function AddingWork() {
...
+                Mathematics               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Biology"}>
+                Biology               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Chemistry"}>
+                Chemistry               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Psychics"}>
+                Psychics
```
Comment (upsu90) [inline web-client/src/pages/adding-work.js]: Physics

**Review by upsu90 — CHANGES_REQUESTED** [review]
placeholder should contain "Select"

**Review by KamilDonda — COMMENTED** [review]
+ image

+ when I choose "Select" I don't get an error: 
image



//EDIT:

`value={""}` -> `value={"Select"}` fixes this error

**Review by KamilDonda — APPROVED** [review]

**Review by upsu90 — APPROVED** [review]

**Review by JustD4nTe — APPROVED** [review]

**Pull request merged**